google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/FieldValue.java

```diff
@@ -191,6 +194,21 @@ public long getTimestampValue() {
     return scaled.longValue();
   }
 
+  /**
+   * Returns this field's value as a {@code String}, representing a timestamp as a readable string.
+   * This method should only be used if the corresponding field has {@link
+   * LegacySQLTypeName#TIMESTAMP} type.
+   *
+   * @throws ClassCastException if the field is not a primitive type
+   * @throws NumberFormatException if the field's value could not be converted to {@link Long}
+   * @throws NullPointerException if {@link #isNull()} returns {@code true}
+   */
+  @SuppressWarnings("unchecked")
+  public Instant getReadableTimestampValue() {
```

Comment:
The name of this getter and the docstring likely need to be revisited.  getTimestampInstant maybe?

Reply:
done.